packages/contentful/src/crn/types/webhook.ts

```diff
@@ -0,0 +1,88 @@
+export interface ContentfulWebhookPublishPayload<T extends 'teams' | 'news'> {
+  metadata: {
+    tags: string[];
+  };
+  sys: {
+    type: 'Entry';
+    id: string;
+    space: {
+      sys: {
+        type: 'Link';
+        linkType: 'Space';
+        id: string;
+      };
+    };
+    environment: {
+      sys: {
+        id: string;
+        type: 'Link';
+        linkType: 'Environment';
+      };
+    };
+    contentType: {
+      sys: {
+        type: 'Link';
+        linkType: 'ContentType';
+        id: T;
+      };
+    };
+    createdBy: {
+      sys: {
+        type: 'Link';
+        linkType: 'User';
+        id: string;
+      };
+    };
+    updatedBy: {
+      sys: {
+        type: 'Link';
+        linkType: 'User';
+        id: string;
+      };
+    };
+    revision: number;
+    createdAt: string;
+    updatedAt: string;
+  };
+  fields: {
+    [key: string]: {
+      'en-US': unknown;
+    };
+  };
+}
+
+export interface ContentfulWebhookUnpublishPayload<T extends 'teams' | 'news'> {
+  sys: {
+    type: 'DeletedEntry';
+    id: string;
+    space: {
+      sys: {
+        type: 'Link';
+        linkType: 'Space';
+        id: string;
+      };
+    };
+    environment: {
+      sys: {
+        id: string;
+        type: 'Link';
+        linkType: 'Environment';
+      };
+    };
+    contentType: {
+      sys: {
+        type: 'Link';
+        linkType: 'ContentType';
+        id: T;
+      };
+    };
+    revision: number;
+    createdAt: string;
+    updatedAt: string;
+    deletedAt: string;
+  };
+}
+
+export type ContentfulWebhookPayload<T extends 'teams' | 'news'> =
+  | ContentfulWebhookPublishPayload<T>
+  | ContentfulWebhookUnpublishPayload<T>;
```

Comment:
It seems like there's possible value in pulling `'teams' | 'news'` out into a `type ContentfulWebhookContentType` since it's a list we know we're going to want to add to over time, and so being able to do so once in a single place might be worthwhile.